packages/design-system-react-native/src/primitives/ButtonBase/ButtonBase.types.ts

```diff
@@ -0,0 +1,95 @@
+import type { PressableProps, StyleProp, ViewStyle } from 'react-native';
+import type { TextProps } from 'src/components/Text/Text.types';
+
+import type { IconProps, IconName } from '../../components/Icon';
+import type { SpinnerProps } from '../../temp-components/Spinner';
+
+export enum ButtonBaseSize {
+  /**
+   * Represents a small button size (32px).
+   */
+  Sm = '32',
+  /**
+   * Represents a medium button size (40px).
+   */
+  Md = '40',
+  /**
+   * Represents a large button size (48px).
+   */
+  Lg = '48',
+}
```

Comment:
non-blocking: What is the reason to use strings instead of numbers here? Using numbers may be more appropriate for size values like this and it would prevent type coercion.